From bd0e239406a7c94da7dfd4c5fa7a4b3785d11c9c Mon Sep 17 00:00:00 2001
From: Alexei Cazacov <alexei.cazacov@qt.io>
Date: Mon, 13 Oct 2025 11:05:59 +0300
Subject: [PATCH 400/553] Doc: Fix instructions on how to use GuiPrivate

This commit fixes the cmake instructions for the GuiPrivate use.

Fixes: QTBUG-141106
Change-Id: Ibb99ef4137e6fa5b394ccf4a33f8e080ca18dcbd
Reviewed-by: Maycon Stamboroski <maycon.stamboroski@qt.io>
Reviewed-by: Joerg Bornemann <joerg.bornemann@qt.io>
(cherry picked from commit 862d323e7d075be5fc97af1fc0f001f3a14a61b0)
Reviewed-by: Qt Cherry-pick Bot <cherrypick_bot@qt-project.org>
---
 src/gui/doc/src/qtgui.qdoc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/gui/doc/src/qtgui.qdoc b/src/gui/doc/src/qtgui.qdoc
index dca2f10bf26..a89872e70f6 100644
--- a/src/gui/doc/src/qtgui.qdoc
+++ b/src/gui/doc/src/qtgui.qdoc
@@ -27,7 +27,7 @@
     private Qt GUI APIs:
 
     \badcode
-    find_package(Qt6 REQUIRED COMPONENTS Gui)
+    find_package(Qt6 REQUIRED COMPONENTS GuiPrivate)
     target_link_libraries(mytarget PRIVATE Qt6::GuiPrivate)
     \endcode
 */
-- 
2.51.2

